iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/GeneratorModification.java

```diff
@@ -170,5 +185,13 @@ public Boolean getConnected() {
         public void setConnected(Boolean connected) {
             this.connected = connected;
         }
+
+        public Boolean getDoNotRestrictTargetPAndDoNotConnect() {
+            return doNotRestrictTargetPAndDoNotConnect;
```

Comment:
This name is quite complex.
I would change it for a more general one (i.e. `ignoreCorrectiveOperations`) and provide javadoc comments with further details on its behavior.

Reply:
fixed:

```
        /**
         * If false and a modification on target P is requested then, the generator is connected (if not before
         * and if it is not contrary to another connected modification), and the target P is set within the generator
         * boundaries (so even after a target P only modification, the generator can have a different connected state
         * and its target P can be its min or max P and not the target P of the modification).
         * If true the modification will simply apply the new target P.
         */
        public boolean getIgnoreCorrectiveOperations() {
            return ignoreCorrectiveOperations;
        }
```

Reply:
To follow the naming conventions, this method should be renamed as `isIgnoreCorrectiveOperations()`.

Reply:
done